src/phoenix/session/session.py

```diff
@@ -47,6 +48,7 @@
 from phoenix.trace.dsl.query import SpanQuery
 from phoenix.trace.trace_dataset import TraceDataset
 from phoenix.utilities import query_spans
+from phoenix.utilities.span_store import get_span_store, load_traces_data_from_store
```

Comment:
this module seems specific to storage. I think it makes sense to nest under store rather than fanning out utilities?

Reply:
There's circular import going on with these types of helper functions because they're a convergence of methods from several different modules. So fanning these out into a "neutral" location is the workaround for now.

Reply:
Ah gotcha. didn't realize it was imported from multiple places. makes sense